app/models/invoice.rb

```diff
@@ -6,5 +6,5 @@ class Invoice < ApplicationRecord
   has_many :invoice_items
   has_many :items, through: :invoice_items
 
-
+  validates :status, presence: true #inclusion:{in: ["cancelled", "completed", "in progress"]}
```

Comment:
Is this the syntax we settled on? I thought we had mentioned something about inclusion_of :status, in [array] and then something similar for the test